### ISSUE TITLE
Force a light theme by default

### ISSUE THREAD

**Issue by obones** [SEPIA-Framework/sepia-docs#147]
**Is your feature request related to a problem? Please describe.**
I have an eye condition that makes any dark theme very difficult to read, which I wrote about [here](https://www.obones.com/en/more/blog/astigmatism-part-1).
SEPIA is in dark theme by default which I can understand but I believe there should be a choice.

**Describe the solution you'd like**
There should be a light theme for SEPIA, or at least a way to invert black background and white text.

**Describe alternatives you've considered**
I could not find any

**Comment by fquirin** [comment]
Hi @obones ,

are you talking about the DIY client (with display) or the regular app/desktop browser page?
Because the default for app and browser page is "use system color" meaning it will only be dark if your mobile or browser is set to dark-mode by default. If this is not the case please let me know your exact OS + browser type and version so I can check for bugs.
The DIY client is indeed set to dark-mode by default because it reads the settings.js file by default and cannot adapt to any user preference.

That said there is a larger number of skins available on any platform. Here are a few [screenshots](https://github.com/SEPIA-Framework/sepia-html-client-app/tree/master/screenshots).
On your mobile/desktop browser/app simply open SEPIA and go to `settings -> general -> skins` to select one that works best for you.
For the DIY client look for `~/clexi/www/sepia/settings.js` and check the field `"activeSkin"`.

Here is the current list with IDs:
```
"Neo Sepia": 			  1
"Neo Sepia Dark": 		  2
"Legacy": 			  3
"Legacy Dark":			  4
"Bytemind":			  5
"Street":			  6
"Sakura":			  7
"9 to 5":			  8
"Autum":			  9
"Nippon":			 10
"Poppy":			 11
"Orange Style 1":		 12
"Orange Style 2.0":		 37
"My Message":			 13
"Uplink":			 14
"Tasty":			 15
"The Grid":			 16
"Canny": 			 17
"Essential": 			 18
"Essential Green": 		 35
"Essential Orange":		 36
"Study": 			 19
"Professional": 		 20
"Odyssey Explorer":		 21
"Odyssey Conqueror":		 22
"Malachite": 			 23
"Nightlife": 			 24
"Spots": 			 25
"Dark Canary": 			 26
"ILA Legacy": 			 27
"Historic Future": 		 28
"Flaming Squirrel":		 30
"Flaming Dark Squirrel":	 31
"Alabaster Mystical":		 32
"Alabaster Motley":		 33
"Alabaster Dark":		 34
```

Let me know if you found one that works for you or if you have some custom design you'd like to see :slightly_smiling_face:

**Comment by obones** [comment]
> are you talking about the DIY client (with display) or the regular app/desktop browser page?

I'm talking about the page that is reached at this URL: http://server:20721/tools/index.html
This is the first one that I tried after having setup SEPIA and I am getting this with Firefox under Win10 x64:
![image](https://user-images.githubusercontent.com/1314739/147605839-521ce329-3bc6-40fc-9142-ac459be3a57d.png)

This one, I could not find a way to change it via settings, and my OS definitely is using light theme. Playing with the CSS directly in Firefox, I got this, which I find very pleasing:
![image](https://user-images.githubusercontent.com/1314739/147606916-f03b54dc-57b9-4083-b0b8-861bc19d4c61.png)

Changing the theme would be nice if it applied at the very start, and that there would be a very simple one without any fancy background.

**Comment by fquirin** [comment]
Oh you mean the Control HUB :sweat_smile: , yes, there is no skin support yet :-|.
I'll add a light skin and a toggle for the next update :slightly_smiling_face: 

Would buttons like this work for you or is the font too bright?

![image](https://user-images.githubusercontent.com/18550513/147668320-86ca2ad7-688f-4f82-bb6d-0a270bc66b23.png)

![image](https://user-images.githubusercontent.com/18550513/147668372-41342d75-133b-449e-969e-c061ea757de6.png)

**Comment by obones** [comment]
> Oh you mean the Control HUB 😅 , yes, there is no skin support yet :-|.
I'll add a light skin and a toggle for the next update 🙂

Ah that's the name of that page, sorry, I have not figured out that just yet.

> Would buttons like this work for you or is the font too bright?

If I was to choose, I'd rather have the while text on light blue background as in the bottom image, but the green ones are acceptable (to me, that is) if there not too many of them

**Comment by fquirin** [comment]
What do you think of this? :slightly_smiling_face: (trying to find a good compromise between original color-scheme and contrast)

![image](https://user-images.githubusercontent.com/18550513/147688536-bb2720ab-6a86-4166-9ebf-b90b1bb76442.png)

**Comment by obones** [comment]
This looks very nice !

**Comment by fquirin** [comment]
The new theme is now available via [SEPIA-Home v2.6.1](https://github.com/SEPIA-Framework/sepia-installation-and-setup/releases) :slightly_smiling_face:
